src/model/channel.rs

```diff
@@ -282,6 +282,7 @@ impl Embed {
     }
 }
 
+/// A group DM.
```

Comment:
This should be added to `group.yml`'s `description`. Although that key is typo'd in the file, if you change that to `description` from `definition` it should have a brief description of what a group is.